File: Data Structures & Algorithms/sum-of-all-subset-xor-totals/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    subsetXORSum(nums) {
        let orValue = 0;

        for (const num of nums) {
            orValue |= num;
        }

        return orValue * (2 ** (nums.length - 1));
    }
}

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-1.py
class Solution:
    def findMin(self, nums: List[int]) -> int:
        if nums[0]<nums[-1]:
            return nums[0]
        else:
            i=0
            while nums[i]>nums[len(nums)-1-i]:
                i+=1
            return min(nums[i],nums[min(len(nums)-1,len(nums)-i)])